packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -959,6 +959,32 @@ export const components = ({
       html {
         font-size: calc((${themeParam.typography.fontSize} / 16) * 100%);
       }
+
+      :not(code) &, :not(pre) & {
+          :lang(el) {
```

Comment:
Just curious, what's `lang(el)`? Also, doesn't setting "Noto Sans" do all of Noto, or is it a variant?

Reply:
`el` is the language code for Greek. Noto Sans is just one font family.

Reply:
There is no Greek specific version of Noto Sans. All we are doing is saying if the language code is Greek, try and use "Noto Sans", we are not importing it into Odyssey.